source/release-notes/3-1-6.rst

```diff
@@ -0,0 +1,35 @@
+.. index:: Crafter CMS 3.1.6 Release Notes
+
+-----------------
+Crafter CMS 3.1.6
+-----------------
+
+^^^^^^^^^^^^
+New Features
+^^^^^^^^^^^^
+
+* Added workflow / publishing support for S3 bucket storage. 
+* Added support for Dashboard APIs that enables content retrieval based on content and metadata criteria defined by users.
```

Comment:
Read more about it: https://app.swaggerhub.com/apis/craftercms/studio/#/dashboard